prussia_rt/src/exceptions/v_common/tlb_modified.rs

```diff
@@ -0,0 +1,33 @@
+use prussia_debug::println_ee;
+
+use crate::{cop0::CoP0Dump, thread::ThreadControlBlock};
+
+/// Purposefully trigger a TLB Modified exception.
+pub unsafe fn trigger_tlb_modified_exception() {
+    println_ee!("Triggering TLB Modified exception.");
+
+    core::arch::asm!("
+        # First prepare a TLB entry by loading a dummy address.
+        lui $k0, 0x1234
+        lw  $k1, 0($k0)
+
+        # Next, update the TLB entry and then write to the virtual address.
+        # This should trigger the exception.
```

Comment:
This testcase seems incomplete in a number of ways. 

The Sony BIOS doesn't dynamically load mappings into the TLB; giant sections of address space get mapped at the start of boot. So the load of a dummy address won't actually cause any TLB activity. I'm not entirely sure *what* gets loaded here; 0x12xx_xxxx is GIF address space, and I think it might just be mirrored.

Anyway, then you load and then re-write the TLB entry as-is. This seems incorrect to me, because the condition for a TLB Modified exception should be that a TLB entry exists for an access, is Valid, but not Dirty; I don't think you're ensuring those conditions for the exception to fire.